unyt/mpl_interface.py

```diff
@@ -164,13 +173,21 @@ def label_style(self, label_style="()"):
         def __enter__(self):
             registry[unyt_array] = unyt_arrayConverter()
             registry[unyt_quantity] = unyt_arrayConverter()
+            self._enabled = True
 
         def __exit__(self, exc_type, exc_val, exc_tb):
             registry.pop(unyt_array)
             registry.pop(unyt_quantity)
+            self._enabled = False
 
         def enable(self):
             self.__enter__()
 
         def disable(self):
-            self.__exit__(None, None, None)
+            if self._enabled:
+                self.__exit__(None, None, None)
+
+        @property
+        def enabled(self):
+            """Return the enable state"""
+            return self._enabled
```

Comment:
Why does this need to be added? Is there something broken about the implementation that got merged last night?

Reply:
I like to use matplotlib_support as a toggle in an interactive session where I may not remember or be able to see easily which state it is in. I could just call matplotlib_support.enable().